src/main/java/net/minecraftforge/event/AbstractForgeEventFactory.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.event;
+
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.eventbus.api.Cancelable;
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraftforge.fml.ModLoader;
+import net.minecraftforge.fml.event.IModBusEvent;
+import org.jetbrains.annotations.ApiStatus;
+
+@ApiStatus.Internal
+public abstract class AbstractForgeEventFactory {
```

Comment:
I didnt wanna make this a common class for the 2 minor helpers because this makes it a public API class modders can use. Yes its marked as internal, but its a whole other class for what is essentially a macro

Reply:
yup